packages/components/src/Button/Button/_docs/Button.stickersheet.stories.tsx

```diff
@@ -251,6 +251,14 @@ const StickerSheetTemplate: StickerSheetStory = {
       </>
     )
   },
+  parameters: {
+    pseudo: {
+      hover: ".story__button--hover",
+      active: ".story__button--active",
+      focus: ".story__button--focus",
+      focusVisible: ".story__button--focus",
```

Comment:
Would these work as data attributes at a global level?

Reply:
These are all temporary styles which get added to the stylesheet when visiting the story. They don't actually exist.
But yes I suppose we could probably use a data attribute as the target (haven't tried).

I think they do become global though, which I'm not sure if it will start meshing the styles as you navigate through stories (they only reset on refresh), which is why I made them unique per stickersheet to ensure nothing weird happens.

Reply:
As discussed, didn't apply it at a global level to avoid confusion with magic data attributes (at least for now), especially since the structure of some of our components pass data attributes into an inner element as opposed to the container (classNameOverride has been put here in most cases though).

Have updated to use data attributes where possible though :)